### PR TITLE
Adds getUsers task

### PR DESCRIPTION
**Pull request by DarsshanP**

### CODE REVIEW

File: __tests__/app.test.js [hunk 1]
```diff
@@ -27,6 +27,16 @@ describe("GET-/api/categories", () => {
         });
       });
   });
+  describe("404: Route not found", () => {
```
Comment (JVal-NC) [inline __tests__/app.test.js:30]: Id suggest these are a bit unnecessary, assume you have 1 test for invalid path already which will cover this and the similar test for api/review. Maybe leave them in for now but don't feel the need to add them going forward.

File: __tests__/app.test.js [hunk 2]
```diff
@@ -268,10 +288,27 @@ describe.only("PATCH /api/reviews/:review_id", () => {
   });
 });
 
-describe("404: Route not found", () => {
+describe.only("GET /api/users", () => {
+  test("GET - 200: Responds with an array of user data", () => {
+    return request(app)
+      .get("/api/users")
+      .expect(200)
+      .then(({ body }) => {
```
Comment (JVal-NC) [inline __tests__/app.test.js:296]: Should be attached to a key, such as body.users

File: controllers/users.controllers.js [hunk 3]
```diff
@@ -0,0 +1,9 @@
+const { fetchUsers } = require("../models/model");
+
+exports.getUsers = (req, res, next) => {
+  fetchUsers()
+    .then((users) => {
+      res.send(users);
```
Comment (JVal-NC) [inline controllers/users.controllers.js]: modify to attach users to response body key of users, as per the test comment

**Review by JVal-NC — COMMENTED** [review]
Few minor comments for inclusion.

**Review by callumu21 — APPROVED** [review]
Good to merge 👍

**Pull request merged**